lib/hammer_cli_katello/repository.rb

```diff
@@ -70,6 +70,10 @@ class InfoCommand < HammerCLIKatello::InfoCommand
           field :docker_manifest_total, _("Docker Manifests"), Fields::Field, :hide_blank => true
           field :docker_tag_total, _("Docker Tags"), Fields::Field, :hide_blank => true
         end
+        collection :ostree_branches, _("RPM OS Tree Branches"),
```

Comment:
I think that would be too long? If people want to get that info, they should use the ostree-branch command.